command_server/command_server.py

```diff
@@ -1,12 +1,13 @@
 import asyncio
-import aiohttp
 import configparser
 import logging
+import aiohttp
 
-from aiohttp import server
+from aiohttp import server, web
 from aiohttp.multidict import MultiDict
-from routes import ROUTES
+from generic import routes
 from urllib.parse import urlparse, parse_qsl
+import views
 
 
 class HttpRequestHandler(aiohttp.server.ServerHttpProtocol):
```

Comment:
If you really need common handle each request, you can inherit from `aiohttp HttpRequestHandler` and configure `web.Application` to use custom one